<!--
%\VignetteEngine{knitr::knitr}
%\VignetteIndexEntry{Salmon quantifications for condition OCT4 knockdown in mouse ESCs}
-->

# Experiment and sample description

This package provides the output of running Salmon on a set of 12
RNA-seq samples from King & Klose, "The pioneer 
factor OCT4 requires the chromatin remodeller BRG1 to support 
gene regulatory element function in mouse embryonic stem cells"
published in eLIFE, March 2017
[doi: 10.7554/eLife.22631](https://doi.org/10.7554/eLife.22631).

All 12 samples from the experiment were re-quantified, which consists
of three replicates each of OCT4 conditional cells treated with 1
micrograms/mL doxycycline for 24 hr and controls (see paper Materials and
Methods), and Brg1 conditional cells treated with 4-hydroxytamoxifen
for 72 hr and controls (see paper Materials and Methods).

# Salmon quantification

[Salmon](https://combine-lab.github.io/salmon/)
version 0.12.0 was run using 
[Gencode](https://www.gencodegenes.org/)
mouse reference transcripts M20, with a 
[snakemake](https://snakemake.readthedocs.io/en/stable/) 
script that can be seen in `inst/scripts/Snakemake` and with 
log output in `inst/scripts/snakemake.log`. 

The quantification output is in the `inst/extdata/quant`
directory. The `quant.sf` files have been gzipped to preserve space,
so they are stored as `quant.sf.gz`. There are 20 Gibbs inferential
replicates for each sample.

# Sample information

The `inst/extdata/coldata.csv` file was obtained from the
`SraRunTable.txt` downloaded directly from SRA. This file contains the
phenotypic information about the 12 samples: 

```{r}
dir <- system.file("extdata", package="oct4")
coldata <- read.csv(file.path(dir,"coldata.csv"))
coldata
```

# Acknowledgments 

Thanks to the study authors for posting their data publicly and
clearly labelling their data, and for James Ashmore for pointing me to
this dataset.

# Session info

```{r}
sessionInfo()
```
